### ISSUE TITLE
"somewhat close to" should occur only with staccato and light green feedback

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#206]
@terracoda mentioned that these two triggers are slightly off. All these feedbacks should occur right at the same time.

**Comment by zepumph** [comment]
@Matthew-Moore240 mentioned that this behavior is working correctly for single hand interaction, so this may just be an issue with "both hands".

**Comment by zepumph** [comment]
It was indeed a small bug. Basically a "greater than or equal to" should have been "greater than". @terracoda please confirm that you cannot reproduce the "somewhat close to ratio"  region when the staccato sounds are not playing (I couldn't). Also a reminder about `?log` which prints the ratio region and some model values along with it.

**Comment by terracoda** [comment]
@zepumph, you got it sorted, thank you! And next time I will post an issue sooner when I first notice it :-)
